backend/src/Manager/Subscription/SubscriptionHistoryManager.php

```diff
@@ -34,4 +35,18 @@ public function createSubscriptionHistory($subscription):SubscriptionHistoryEnti
  
         return $subscriptionHistoryEntity;
     }
+
+    public function updateNoteSubscriptionHistory($subscriptionId, $note):SubscriptionHistoryEntity
+    { 
+        $subscriptionHistoryEntity = $this->subscribeHistoryRepository->findOneBy(["subscription" => $subscriptionId]);
+      
+        $subscriptionHistoryEntity->setNote($note);
+       
+        $subscriptionHistoryEntity->setCreatedAt(new DateTime());
+        
+        $this->entityManager->persist($subscriptionHistoryEntity);
```

Comment:
If the function updates field/s, why using _persist()_?

Reply:
done